src/services/redisSessionStorage.ts

```diff
@@ -432,8 +417,40 @@ export default class RedisSessionStorage extends RedisStorageUtils
             _ !== `${sessionInfoKeyPrefix}${user.session_token}`
         )
         .map(_ => `${sessionKeyPrefix}${_.split(sessionInfoKeyPrefix)[1]}`);
+      /**
```

Comment:
please restrict this comment style to function description : )

Reply:
fixed in 276bafa